packages/connect-react/src/hooks/form-field-context.tsx

```
@@ -0,0 +1,31 @@
+import {
+  createContext, useContext,
+} from "react";
+import type {
+  AppInfo, ConfigurableProp, ConfigurablePropApp, PropValue,
+} from "@pipedream/sdk";
+
+export type FormFieldContextExtra<T extends ConfigurableProp> = T extends ConfigurablePropApp ? {
+  app?: AppInfo;
+} : {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)